tests/e2e/doctests/002-pip-binary-installation-test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2025 Huawei Technologies Co., Ltd. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# This file is a part of the vllm-ascend project.
+#
+set -euo pipefail
+
+trap clean_venv EXIT
+check_npus
+create_vllm_venv
+
+_info "====> Install vllm and vllm-ascend from ${VLLM_VERSION}"
```

Comment:
FYI: https://github.com/vllm-project/vllm-ascend/pull/1179

1. only check current branch version is enough
2. we should get version from doc conf.py, we don't want to do anything when bump version